### PR TITLE
docker dev and test setup fixes

### PR DESCRIPTION
**Pull request by machisuji**

### CODE REVIEW

File: spec/support/browsers/chrome.rb [hunk 1]
```diff
@@ -45,20 +45,25 @@ def register_chrome(language, name: :"chrome_#{language}")
     client.read_timeout = 180
     client.open_timeout = 180
 
-    service = ::Selenium::WebDriver::Service.chrome(args: { verbose: true, log_path: '/tmp/chromedriver.log' })
+    is_grid = ENV['SELENIUM_GRID_URL'].present?
```
Comment (b12f) [inline spec/support/browsers/chrome.rb:48]: We might want to align `firefox.rb` to be in the same style

**Review by b12f — COMMENTED** [review]
lgtm

File: bin/compose [hunk 2]
```diff
@@ -15,13 +15,25 @@ if [ $# -eq 0 ]; then
   echo
   echo "Commands:"
   echo "    setup - Has to be run once initially. Installs backend and frontend dependencies. "
+  echo "    reset - Resets everything by removing all containers and deleting all volumes. You need to run \`setup\` again afterwards. "
   echo "    start - Starts both backend and frontend in the background. Access via http://localhost:3000/ by default."
   echo "    run   - Starts the frontend in the background and backend in the foreground. Useful for debugging using pry."
+  echo "    rspec - Runs rspec inside the \`backend-test\` container which will be started it it's not running yet."
```
Comment (tessi) [inline bin/compose]:
```suggestion
  echo "    rspec - Runs rspec inside the \`backend-test\` container which will be started if it's not running yet."
```

File: docker-compose.yml [hunk 3]
```diff
@@ -143,7 +143,8 @@ services:
       CAPYBARA_SERVER_PORT: 3000
       CAPYBARA_DYNAMIC_BIND_IP: 1
       CAPYBARA_APP_HOSTNAME: backend-test
-      OPENPROJECT_DISABLE_DEV_ASSET_PROXY: 1
+      # OPENPROJECT_DISABLE_DEV_ASSET_PROXY: 1
```
Comment (tessi) [inline docker-compose.yml]:
```suggestion
```

There is probably no need for this being around, right?

**Review by tessi — COMMENTED** [review]
not a real review (as I'm too far away from grasping all needed details), just two minor comments :)
👋

**Comment by machisuji** [comment]
> not a real review (as I'm too far away from grasping all needed details), just two minor comments :)
> 👋

Thanks! 👍

File: docs/development/development-environment-docker/README.md [hunk 4]
```diff
@@ -21,6 +21,16 @@ bin/compose start
...
 
+### Tests
+
+You can run tests using `bin/compose rspec`. You can run specific tests too. For instance:
+
+```
+bin/compose rspec spec/features/work_package_show_spec.rb
+```
+
+***
```
Comment (ulferts) [inline docs/development/development-environment-docker/README.md:32]: I'd place this line above the section about tests. 

Additionally, I'd take the chance to rename the heading `Setup` to something like `Setup (Step by step)` to be easily distinguishable from `Quickstart`

File: docs/development/development-environment-docker/README.md [hunk 5]
```diff
@@ -21,6 +21,16 @@ bin/compose start
...
+
 If there is an `.env` file (see below) `bin/compose` will source it.
 More details and options follow in the next section.
```
Comment (ulferts) [inline docs/development/development-environment-docker/README.md:34]: This line might also be moved to within the `Setup` section.

**Review by ulferts — APPROVED** [review]
Works on my machine. I have only some suggestions on improving the documentation.

**Pull request merged**